t/09_qesapdeployment.t

```diff
@@ -789,4 +789,95 @@ subtest '[qesap_prepare_env::qesap_create_aws_config] not solved template and va
     like $contents[0], qr/region = eu-central-1/, "Expected region eu-central-1 is in the config file";
 };
 
+subtest '[qesap_is_job_finished]' => sub {
+    my $qesap = Test::MockModule->new('qesapdeployment', no_auto => 1);
+    my @results = ();
+    $qesap->redefine(script_output => sub {
+            if ($_[0] =~ /100000/) { return "not json"; }
+            if ($_[0] =~ /200000/) { return "{\"job\":{\"state\":\"donaldduck\"}}"; }
+            if ($_[0] =~ /300000/) { return "{\"job\":{\"state\":\"running\"}}"; }
+    });
+
+    $qesap->redefine(get_required_var => sub { return ''; });
+    $qesap->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', @_)); });
+
+    push @results, qesap_is_job_finished(100000);
+    push @results, qesap_is_job_finished(200000);
+    push @results, qesap_is_job_finished(300000);
+
+
+    ok($results[0] == 0, "Consider 'running' state if the openqa job status response isn't JSON");
+    ok($results[1] == 1, "Considered 'finished' state if the openqa job status response exists and isn't 'running'");
+    ok($results[2] == 2, "Consider 'running' if the openqa job status response is 'running'");
+};
+
+subtest '[qesap_az_get_active_peerings]' => sub {
```

Comment:
This is a test about a qesap_az function (qesap function composing az commands): please move this test to t/15_qesap_azure.t